apps/mobile-wallet/src/navigation/RootStackNavigation.tsx

```diff
@@ -72,7 +71,7 @@ import { mnemonicMigrated, walletUnlocked } from '~/store/wallet/walletActions'
 import { showExceptionToast, showToast } from '~/utils/layout'
 import { resetNavigation, rootStackNavigationRef } from '~/utils/navigation'
 
-const RootStack = createStackNavigator<RootStackParamList>()
+const RootStack = createNativeStackNavigator<RootStackParamList>()
```

Comment:
I'll just leave a comment here so that we keep track of the Slack discussion

## This PR

https://github.com/user-attachments/assets/9eca8074-0fd6-4cbf-b6c0-701a01b8637f


## Prod (master)

https://github.com/user-attachments/assets/36865dee-a1fb-4cda-8110-2481d9332c86

Reply:
Fixed in follow up PR!